src/js/app.js

```diff
@@ -820,11 +847,10 @@ app.controller('walletRecoveryCtrl', function($scope, $q, $modal, $rootScope, $l
                         }
 
                         return sdk.client.post(
-                            "/wallet/" + $scope.backupDataV2.walletIdentifier + "/signbcc",
+                            "/wallet/" + $scope.backupDataV2.walletIdentifier + "/" + window.APPCONFIG.COSIGN_ENDPOINT,
```

Comment:
hmm, when I search the diff and the appconfig.json, the value isn't set. does it need to be set for BTC/BCH?

Reply:
Only if the login functionality is being used.